### PR TITLE
Added test page for Electron app

### PR DESCRIPTION
**Pull request by Nick-McGee**
Created an HTML/CSS page for our Electron app. It still has some non-responsive elements that need to be changed, as well as some flexbox bugs.

### CODE REVIEW

File: minecraftIndexPage/src/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <nav id="main-nav">
+      <a href="index.html" class="btn" id="login">Lessons</a>
+      <img src="img/logo.png" alt="Minecraft Education" id="logo">
+      <a href="login.html" class="btn" id="login">Login</a>
+    </nav>
+
+    <div id="lessons">
+      <div class="lesson">
+        <h1 class="lesson-h1">Lesson 1</h1>
+        <img src="img/lesson1.png" alt="lesson1" class="lesson-img">
```
Comment (Ansper11) [inline minecraftIndexPage/src/index.html:19]: That looks really cool

**Comment by Ansper11** [comment]
It is a good start thanks Nick

**Comment by eloiseespel** [comment]
This is nice, thanks for posting an example!

**Comment by eloiseespel** [comment]
Do you know what all the ttf files are for?

**Comment by Nick-McGee** [comment]
> Do you know what all the ttf files are for?

They are the font files for the Minecraft looking font.

**Pull request merged**